scripts/irods/test/test_resource_types.py

```diff
@@ -3678,17 +3688,21 @@ def tearDown(self):
 
     @unittest.skipIf(test.settings.RUN_IN_TOPOLOGY, "local filesystem check")
     def test_ichksum_no_file_modified_under_replication__4099(self):
+        #
+        # TODO IS THIS TEST NEEDED ANYMORE? WHAT DOES THIS TEST MEAN?
+        #
+
```

Comment:
Do we need this test?

This test does not appear to be useful because the checksum API detects the situation and returns an error. Notice the changes to the test. The reason why this test worked before was because the checksum API read the entire file on disk rather than the data size in the catalog.

Reply:
same as above - this case should no longer need a test.

Reply:
See comments above.